dataplane.code-workspace

```diff
@@ -157,6 +157,7 @@
       "**/*.d.ts": true,
       "**/test-browser/*": true
     },
+    "typescript.tsdk": "core-http\\node_modules\\typescript\\lib",
```

Comment:
This is neat, so this makes vscode use whatever version of TypeScript is installed in core-http then, right?

Reply:
yes! This way VS Code should always match what we're building with